15LessonFifteen/Checkpoint3/css/style.css

```diff
@@ -0,0 +1,573 @@
+
+
+.page  {
+  display: flex;
+  flex-direction: column;
+}
+
+h1 {
+  text-align: center;
+}
+
+p {
+  text-align: center;
+}
+
+img {
+  max-width: 100%;
+  height: 74vh;
+  width: 100%;
+}
+
+article {
+  width: 80%;
+}
+
+.wines3 article img {
+  width: 45%;
+  height: 55vh;
+  margin-top: -3vh;
```

Comment:
These values here are causing the wine bottles to creep up into the section before.